### PR TITLE
Support Strimzi Kafka feature gates

### PR DESCRIPTION
**Pull request by agrski**

### CODE REVIEW

File: roles/strimzi_kafka/defaults/main.yaml [hunk 1]
```diff
@@ -15,3 +15,8 @@ strimzi_kafka_create_prometheus_instance: true
 
 strimzi_kafka_install_grafana: true
 strimzi_kafka_grafana_prometheus_url: http://prometheus-operated.kafka:9090
+
+strimzi_kafka_add_feature_gates: true
+strimzi_kafka_feature_gates:
+  - "+UseKRaft"
```
Comment (RafalSkolasinski) [inline roles/strimzi_kafka/defaults/main.yaml:21]:
```suggestion
- "+UseKRaft"
```

File: roles/strimzi_kafka/defaults/main.yaml [hunk 2]
```diff
@@ -15,3 +15,8 @@ strimzi_kafka_create_prometheus_instance: true
...
+  - "+UseStrimziPodSets"
```
Comment (RafalSkolasinski) [inline roles/strimzi_kafka/defaults/main.yaml:22]:
```suggestion
- "+UseStrimziPodSets"
```

File: roles/strimzi_kafka/defaults/main.yaml (hunk 1 above)
Comment (agrski) [reply roles/strimzi_kafka/defaults/main.yaml:21]: It seems we generally indent lists, e.g. [here](https://github.com/SeldonIO/ansible-k8s-collection/blob/master/playbooks/seldon_core.yaml#L4) and [here](https://github.com/SeldonIO/ansible-k8s-collection/blob/master/playbooks/seldon_core.yaml#L22)

**Comment by agrski** [comment]
Converting to draft while I do some testing

**Pull request merged**